### PR TITLE
[stable/ghost] Change default ghost port

### PR DESCRIPTION
**Pull request by tompizmor**
The ghost chart was wrongly configuring the ghostPort as 80 by default. It should be 2368 by default as it is in the docker image. 
Furthermore, it is not possible to start ghost in the port 80 since the docker image was moved to a non-root approach.

### CODE REVIEW

**Comment by mattfarina** [comment]
@tompizmor Are you saying the current chart is broken and won't run?

**Comment by unguiculus** [comment]
I don't see why this was problematic. All you did is change the default port configuration. It is still possible to configure it back to port 80 again which should still work. Otherwise the whole port configuration would be broken.

**Comment by tompizmor** [comment]
@mattfarina @unguiculus sorry for the late response. 

The current chart is not working for me so the whole port configuration seems to be broken.
As the ghost image run as a non-root user it is not possible to run Ghost in the port 80. 

The chart worked before because there was a bug in the Ghost application that was ignoring the port in the config file and always using 2368.

File: stable/ghost/templates/NOTES.txt [hunk 1]
```diff
@@ -36,10 +34,17 @@ host. To configure Ghost with the URL of your service:
...
+
+  echo Blog URL  : http://$APP_HOST:$APP_PORT/
+  echo Admin URL : http://$APP_HOST:$APP_PORT/ghost
+
 {{- else }}
 
-  echo Blog URL  : http://{{ include "ghost.host" . }}:{{ default "80" .Values.ghostPort }}/
-  echo Admin URL : http://{{ include "ghost.host" . }}:{{ default "80" .Values.ghostPort }}/ghost
+  echo Blog URL  : http://{{ include "host" . }}/
+  echo Admin URL : http://{{ include "host" . }}/ghost
```
Comment (sameersbn) [inline stable/ghost/templates/NOTES.txt]: `http://{{ include "host" . }}/` -> `http://{{ include "ghost.host" . }}/`

**Review by sameersbn — CHANGES_REQUESTED** [review]
@tompizmor If the issue really is with using the port `80`, shouldn't we instead

1. update the default port here to `2368`
2. clean to up to remove all instances of ` {{ default "80" .Values.ghostPort | quote }}` with simply `"{{ .Values.ghostPort }}"`
3. add login to abort the deployment of user specifies `ghostPort <= 1024`

Alternatively we should hardcode the container port to `2368` and use the `ghostPort` setting to configure the service port.

**Comment by sameersbn** [comment]
@tompizmor 

p.s. my attempts to test the PR changes resulted in failure. Upon enabling the NAMI debugger I see the following:

```
ghost   INFO  ==> Configuring Ghost...
ghost   TRACE [runProgram] Executing: ghost setup,--no-setup-nginx,--no-setup-ssl,--no-setup-mysql,--no-setup-systemd,--no-setup-linux-user,--url,http://10.160.0.2:2368,--ip,0.0.0.0,--port,2368,--log,file,--process,local,--no-prompt,--no-start,--no-enable
ghost   TRACE [runProgram] Executing internal command: ghost ["setup","--no-setup-nginx","--no-setup-ssl","--no-setup-mysql","--no-setup-systemd","--no-setup-linux-user","--url","http://10.160.0.2:2368","--ip","0.0.0.0","--port",2368,"--log","file","--process","local","--no-prompt","--no-start","--no-enable"]
ghost   TRACE [runProgram] RESULT: {"code":130,"stderr":"Terminated\n","stdout":"You are running an outdated version of Ghost-CLI.\nIt is recommended that you upgrade before continuing.\nRun `npm install -g ghost-cli@latest` to upgrade.\n\n? Continue without upgrading? (y/N) \u001b[36D\u001b[36C"}
Error executing 'postInstallation': Terminated

nami    TRACE Error: Error executing 'postInstallation': Terminated

    at runProgram (/opt/bitnami/nami/node_modules/nami-utils/lib/os/run-program.js:223:14)
    at Object.runProgram (/opt/bitnami/nami/node_modules/nami-utils/lib/os/index.js:63:12)
    at Object.$app.helpers.configureGhost (/.nami/components/com.bitnami.ghost/helpers.js:123:7)
    at Service.$app.postInstallation (/.nami/components/com.bitnami.ghost/main.js:44:18)
    at Service.runStep (/opt/bitnami/nami/node_modules/nami-core/lib/components/component.js:159:12)
    at _.each.step (/opt/bitnami/nami/node_modules/nami-core/lib/components/component.js:236:32)
    at ot (/opt/bitnami/nami/node_modules/lodash/index.js:12:507)
    at Function.<anonymous> (/opt/bitnami/nami/node_modules/lodash/index.js:33:421)
    at Service.install (/opt/bitnami/nami/node_modules/nami-core/lib/components/component.js:236:7)
    at Manager.initializePackage (/opt/bitnami/nami/node_modules/nami-core/lib/manager.js:284:11)
```

**Comment by sameersbn** [comment]
@tompizmor 

p.s. my attempts to test the PR changes resulted in failure. Upon enabling the NAMI debugger I see the following:

```
ghost   INFO  ==> Configuring Ghost...
ghost   TRACE [runProgram] Executing: ghost setup,--no-setup-nginx,--no-setup-ssl,--no-setup-mysql,--no-setup-systemd,--no-setup-linux-user,--url,http://10.160.0.2:2368,--ip,0.0.0.0,--port,2368,--log,file,--process,local,--no-prompt,--no-start,--no-enable
ghost   TRACE [runProgram] Executing internal command: ghost ["setup","--no-setup-nginx","--no-setup-ssl","--no-setup-mysql","--no-setup-systemd","--no-setup-linux-user","--url","http://10.160.0.2:2368","--ip","0.0.0.0","--port",2368,"--log","file","--process","local","--no-prompt","--no-start","--no-enable"]
ghost   TRACE [runProgram] RESULT: {"code":130,"stderr":"Terminated\n","stdout":"You are running an outdated version of Ghost-CLI.\nIt is recommended that you upgrade before continuing.\nRun `npm install -g ghost-cli@latest` to upgrade.\n\n? Continue without upgrading? (y/N) \u001b[36D\u001b[36C"}
Error executing 'postInstallation': Terminated

nami    TRACE Error: Error executing 'postInstallation': Terminated

    at runProgram (/opt/bitnami/nami/node_modules/nami-utils/lib/os/run-program.js:223:14)
    at Object.runProgram (/opt/bitnami/nami/node_modules/nami-utils/lib/os/index.js:63:12)
    at Object.$app.helpers.configureGhost (/.nami/components/com.bitnami.ghost/helpers.js:123:7)
    at Service.$app.postInstallation (/.nami/components/com.bitnami.ghost/main.js:44:18)
    at Service.runStep (/opt/bitnami/nami/node_modules/nami-core/lib/components/component.js:159:12)
    at _.each.step (/opt/bitnami/nami/node_modules/nami-core/lib/components/component.js:236:32)
    at ot (/opt/bitnami/nami/node_modules/lodash/index.js:12:507)
    at Function.<anonymous> (/opt/bitnami/nami/node_modules/lodash/index.js:33:421)
    at Service.install (/opt/bitnami/nami/node_modules/nami-core/lib/components/component.js:236:7)
    at Manager.initializePackage (/opt/bitnami/nami/node_modules/nami-core/lib/manager.js:284:11)
```

**Comment by chipironcin** [comment]
```
You are running an outdated version of Ghost-CLI.\nIt is recommended that you upgrade before continuing.\nRun `npm install -g ghost-cli@latest` to upgrade.\n\n? Continue without upgrading? (y/N)
```

I've been dealing with this the past couple of days.
Even if we fix the chart this will keep appearing because Ghost does a live update check.
We should find a way of bypassing it, and has nothing to do with the Chart

**Comment by tompizmor** [comment]
@sameersbn @chipironcin So finally the issue was not on the chart but in the container. I have reverted the changes affecting the port configuration but I keep the improvements in the NOTES file.

The image bitnami/ghost:1.8.3-r0 fixes the port configuration, update node from 6.x to 8.9.x and also fixes the issue when there is a new version of ghost-cli.

The chart should work fine again with this PR.

**Review by sameersbn — APPROVED** [review]

**Pull request merged**